expressions.go

```diff
@@ -100,7 +100,7 @@ func (e tSelector) Compute(ws *Worksheet) (Value, error) {
 	} else if selectedWs, ok := value.(*Worksheet); ok {
 		return tSelector(e[1:]).Compute(selectedWs)
 	} else if selectedSlice, ok := value.(*Slice); ok {
-		// returnedSlice :=newSlice(nil) // TODO(pascal)
+		var elementType Type = &UndefinedType{}
```

Comment:
Even if the slice is empty (has no elements), it seems like the type should still be `elementType = subWs.def.fieldsByName[e[1]].Type()`.

Worth adding to the `TestSelectorSliceTypes` that after you `parent.MustDel(0)`, using the same selector `tSelector([]string{"refs_to_children", "name"})` yields an empty slice with the proper type of `[]text`.

Reply:
Cool, I can do that. Since `subWs` won't exist in that case though we have to grab the type from the definition...something like `var elementType Type = ws.def.fieldsByName[e[0]].Type().(*SliceType).ElementType().(*Definition).fieldsByName[e[1]].Type()`

Reply:
Seems trivial.

Reply:
j/k